sdk/storage/test/test_base.hpp

```diff
@@ -59,4 +61,7 @@ namespace Azure { namespace Storage { namespace Test {
 
   void DeleteFile(const std::string& filename);
 
+  std::string ToISO8601(const std::chrono::system_clock::time_point& time_point);
```

Comment:
nit: `ToIso8601` and `ToRfc1123`?

Reply:
We don't have guideline for that yet. Why not bring this up in the next weekly meeting?